.azure/azure-pipelines.ci.yml

```diff
@@ -253,7 +253,7 @@ stages:
       jobName: wanperf_30Mbps
       displayName: 30 Mbps
       iterations: 3
-      durationMs: 10000
+      length: 37500000
```

Comment:
How were the lengths here chosen?

Reply:
(Rate / 8) * "Previous Duration"

Reply:
I wasn't really trying to change functionality yet.